_stbt/control.py

```diff
@@ -51,6 +51,7 @@ def uri_to_remote_recorder(uri):
          lirc_remote_listen_tcp),
         (r'lirc:(?P<lircd_socket>[^:]+)?:(?P<control_name>.+)',
          lirc_remote_listen),
+        (r'lircd(:(?P<lircd_socket>[^:]+))?', lircd_remote_listen),
```

Comment:
I wondered if this needs to be available to `--contol-recorder` -- it might be less confusing just to have it internally. But then I saw that your `stbt control-relay` can take any of these as inputs, which could be useful I suppose.